ACHNBrowserUI/ACHNBrowserUI/viewModels/ChoreFormViewModel.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  ChoreFormViewModel.swift
+//  ACHNBrowserUI
+//
+//  Created by Otavio Cordeiro on 29.05.20.
+//  Copyright © 2020 Thomas Ricouard. All rights reserved.
+//
+
+import Backend
+import Combine
+
+class ChoreFormViewModel: ObservableObject {
+
+    // MARK: - Properties
+
+    @Published var chore: Chore
+
+    // MARK: - Life cycle
+
+    init(chore: Chore?) {
+        if let chore = chore {
```

Comment:
you could simplify the code here
```swift
self.chore = chore ?? Chore()
```

Reply:
Oops. I overlooked that. Thanks for pointing that out.

Reply:
Done, @renaudjenny.